Sources/Amplitude/Plugins/iOS/UIKitUserInteractions.swift

```diff
@@ -213,13 +213,16 @@ extension UIView: ActionTrackable {
     @objc func amp_shouldTrack(_ action: Selector, for target: Any?) -> Bool { false }
 }
 
-extension UIButton {
-    override var amp_title: String? { currentTitle }
+extension UIControl {
     override func amp_shouldTrack(_ action: Selector, for target: Any?) -> Bool {
         actions(forTarget: target, forControlEvent: .touchUpInside)?.contains(action.description) ?? false
     }
 }
 
+extension UIButton {
+    override var amp_title: String? { currentTitle }
```

Comment:
Would be good to add some fallback to the image name, if available - many buttons will be icon only. Can be a follow up.

Reply:
There seems to be two options to get the image name if `currentTitle` is not available:
```
UIButton.currentImage?.imageAsset?.value(forKey: "assetName") as? String
```
or
```
UIButton.currentImage?.accessibilityIdentifier
```
Any thought on which option to choose?

Reply:
I would use `UIButton.currentImage?.accessibilityIdentifier`. Stick to public APIs where possible for iOS, use of private apis can get flagged in App Store review or the apis themselves can have undocumented changes between OS versions. If you really, really need one, see if it's used in any other major open source projects, which would give some indication of safety with reviews.